environments/key4hep-common/packages.yaml

```diff
@@ -12,13 +12,17 @@ packages:
   gdb:
     require: +python+tui+source-highlight~debuginfod
   # https://github.com/key4hep/key4hep-spack/pull/501
+  geant4:
+    require: +qt+opengl
```

Comment:
Why not be explicit about the no vecgeom here? or does `~vecgeom` not work?

Reply:
Well it has been the default forever, I don't think it matters for this one

Reply:
What is "it" that was the default? 

Can we be explicit if anything ever requires vecgeom in geant for whatever reason, so we get a conflict? Now that we still remember this issue?

Reply:
Done

Reply:
Thanks!